<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdjustItemForm</class>
 <widget class="QWidget" name="AdjustItemForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="listWidgetZoom">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>101</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetAngle">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>101</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Angle</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>391</width>
     <height>391</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>角度範囲</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>最小倍率</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>最大倍率</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>個片サイズ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>個片探索範囲</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxAngleRange">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <double>180.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMinZoomSize">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.200000000000000</double>
   </property>
   <property name="maximum">
    <double>5.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxZoomSize">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.200000000000000</double>
   </property>
   <property name="maximum">
    <double>5.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPieceSize">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>15</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSearchDotPiece">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonRelrectOnlyPalletize">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>470</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>このブロックだけに反映</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonLoadFromLibrary">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>510</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ライブラリから読み出し</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonReflectAllPalletize">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>470</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>同一種類のブロックに反映</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSaveToLibrary">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>512</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ライブラリに保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonClose">
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>510</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>閉じる</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditLibName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>42</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Library Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditLibID">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>12345</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>採用分散値</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxThresholdV">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>最大個片抽出数</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxMaxCountOfPiece">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMasterPosition">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>510</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Master Position</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>決定標準検査しきい値</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxThreshCoeff">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSearchNearBy">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>近傍探索範囲</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
